third_party/move/move-vm/runtime/src/loader/mod.rs

```diff
@@ -2303,17 +2315,26 @@ impl Loader {
             let struct_name = &*self
                 .struct_name_index_map()
                 .idx_to_struct_name(struct_name_idx);
-            return Err(
-                PartialVMError::new(StatusCode::UNKNOWN_INVARIANT_VIOLATION_ERROR).with_message(
-                    format!(
-                        "Depth formula for struct '{}' and formula {:?} (struct type: {:?}) is already cached: {:?}",
-                        struct_name,
-                        formula,
-                        struct_type.as_ref(),
-                        f
-                    ),
-                ),
+
+            // TODO(loader_v2): Revisit tis, because now we do share the VM...
```

Comment:
This seemed to be a bigger problem related to race condition. Should we just fix it in this PR?

Reply:
Probably a PR on top, not the 3K one :)

Reply:
that makes sense and standardizing loader_v2 todos is good.
what about the println and commented out lines below?

Reply:
They are related to the todo? Once VM is shared we will hit this issue, commenting out allows to test things e2e without requiring a fix for that in one of the stacked PRs.